Header header
mobility_msgs/PositionTargetMode target
mavros_msgs/ActuatorControl actuator_control
mavros_msgs/RCOut rcout
nav_msgs/Odometry odom
geometry_msgs/TwistStamped vel_est_encoders
geometry_msgs/WrenchStamped accel
geometry_msgs/Vector3 vel_body
geometry_msgs/Vector3 accel_body
geometry_msgs/Vector3 error_pos
geometry_msgs/Vector3 error_vel
geometry_msgs/Vector3 thrust
float64 roll_desired
float64 pitch_desired
float64 yaw_desired
float64 r_M_y
float64 r_M_z
float64 r_M_x
float64 roll
float64 pitch
float64 yaw
float64 roll_rate
float64 pitch_rate
float64 yaw_rate
float64 throttle
float64 desired_velocity
float64 posI
float64 velI
float64 yawI
float64 pitchI
float64 xI
float64 yI
float64 zI
float64 total_operation_time